test/command_callback/test_cucumber_command_callback.vader

```diff
@@ -0,0 +1,25 @@
+Before:
+  runtime ale_linters/ruby/rubocop.vim
+  call ale#test#SetDirectory('/testplugin/test/')
+
+After:
+  Restore
+
+  call ale#linter#Reset()
+  call ale#test#RestoreDirectory()
+
+Execute(Should require the nearest features dir, if one is found):
+  call ale#test#SetFilename('cucumber_fixtures/features/cuke.feature')
+
+  AssertEqual
+  \   'cucumber --dry-run --quiet --strict --format=json '
+  \   . '-r ''' . ale#path#Simplify('/testplugin/test/cucumber_fixtures/features/') . ''' %t',
```

Comment:
You can fix the problem with Windows tests here by using `g:dir . '/rest-of-path'`. You can use `../` if you need to. That will start the path with the drive letter. Failing that, you could replace `^/` with `C:\` if `has('win32')` is true.